main/src/test/java/org/mobilitydata/gtfsvalidator/validator/UrlConsistencyValidatorTest.java

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright 2021 MobilityData IO
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.validator;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.common.collect.ImmutableList;
+import java.time.ZoneId;
+import java.util.List;
+import java.util.Locale;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mobilitydata.gtfsvalidator.notice.NoticeContainer;
+import org.mobilitydata.gtfsvalidator.notice.ValidationNotice;
+import org.mobilitydata.gtfsvalidator.table.GtfsAgency;
+import org.mobilitydata.gtfsvalidator.table.GtfsAgencyTableContainer;
+import org.mobilitydata.gtfsvalidator.table.GtfsRoute;
+import org.mobilitydata.gtfsvalidator.table.GtfsRouteTableContainer;
+import org.mobilitydata.gtfsvalidator.table.GtfsRouteType;
+import org.mobilitydata.gtfsvalidator.table.GtfsStop;
+import org.mobilitydata.gtfsvalidator.table.GtfsStopTableContainer;
+import org.mobilitydata.gtfsvalidator.validator.UrlConsistencyValidator.SameRouteAndAgencyUrlNotice;
+import org.mobilitydata.gtfsvalidator.validator.UrlConsistencyValidator.SameStopAndAgencyUrlNotice;
+import org.mobilitydata.gtfsvalidator.validator.UrlConsistencyValidator.SameStopAndRouteUrlNotice;
+
+@RunWith(JUnit4.class)
+public class UrlConsistencyValidatorTest {
```

Comment:
All those tests are great. Maybe adding tests to explicitly test the methods creating the URL maps would be good.

Reply:
My bad, the methods creating the URL maps are privates, you can skip the tests as they cannot be called outside the class.